README.md

```diff
@@ -26,14 +26,43 @@ The mod sends a bunch of information about your game to a web API. The API then
 
 If you would like to know exactly what is sent then you can ask in the Discord or look through the code.
 
+### Manual Compilation
+
+*Although this guide is built for IntelliJ, it will work with any IDE you choose as long as you set the java executables properly.*
+
+- Clone the repository and open it up with Intellij.
+- Don't worry! Things will break when you first open it up, this is normal!
+- Go to `File -> Project Structure -> Project` and set the SDK to 1.8. (You can click `Add SDK -> Download JDK` (And that will also work!)
+- Now we need to set the gradle version. Click on the `Gradle` tab (the default is in the top right tab), click the gear, and then hit `Gradle Settings`
+- Now assign the Gradle JVM to JDK 17. Like in step 3, you may use the downloaded Amazon Corretto build.
+- Sync your Gradle tasks by hitting the reload icon and wait for everything to load.
+- Now to build, go back to the Gradle tab and click `Tasks -> Build -> Jar` and in a few minutes you can find the jar in the build folder.
+
+
 ### Contributing
+*Follow everything above for Manual Compilation*
+
+#### Setting up a development environment
+If you've just finished the steps above you should see a `Minecraft Client` task in the top (If you don't you might have to restart IntellIJ first), this gives you a live enviroment for rapid prototyping
+
```

Comment:
I didn't have the task at the top; I'd suggest adding another paragraph to the "Setting up a development environment" section so it looks like this:
```suggestion

If you can't see that task, it may be hidden in your task list: click on the dropdown arrow next to the current task, select "All Configurations", and select the "Minecraft Client" task.

```